Library/Formula/git-octopus.rb

```diff
@@ -0,0 +1,11 @@
+class GitOctopus < Formula
+  homepage "https://github.com/lesfurets/git-octopus"
+  url "https://github.com/lesfurets/git-octopus/archive/v1.0.tar.gz"
```

Comment:
Need a ` sha256 ` line here as well. Non-checksummed formulae won't be accepted.